src/wp-includes/kses.php

```diff
@@ -1102,7 +1102,7 @@ function wp_kses_split2( $content, $allowed_html, $allowed_protocols ) {
 	}
 
 	// It's seriously malformed.
-	if ( ! preg_match( '%^<\s*(/\s*)?([a-zA-Z0-9-]+)([^>]*)>?$%', $content, $matches ) ) {
+	if ( ! preg_match( '%^<\s*(/\s*)?([a-zA-Z0-9-]+:?+[a-zA-Z0-9-]*)([^>]*)>?$%', $content, $matches ) ) {
```

Comment:
I don't think the `+` in `:?+` is necessary. Can you clarify why this was added?

Reply:
You're right. It's not necessary. 
I'll simplify the regex based on https://core.trac.wordpress.org/ticket/34105

Reply:
```suggestion
	if ( ! preg_match( '%^<\s*(/\s*)?([a-zA-Z0-9-:]+)([^>]*)>?$%', $content, $matches ) ) {
```